### ISSUE TITLE
Translate the words of the site need to modify the status

### ISSUE THREAD

**Issue by baiyanghk** [phpList/phplist3#908]
@marianaballa 
I have translated some of the words so far, but there are many words on the translation site that cannot be modified, so I need the administrator to modify the permissions.

They both show that {Read only strings} cannot be modified

[phpList/phpList3 — Chinese (Simplified) @ phpList translate](https://translate.phplist.org/projects/phplist/phplist3/zh_Hans/)

**Comment by michield** [comment]
Yes, how strange. No idea why it does that. @aulona1 or @marianaballa can you find out how to remove the "read only" from those strings?

**Comment by baiyanghk** [comment]
@marianaballa can you find out how to remove the "read only" from those strings?
